share/lib/python/neuron/rxd/region.py

```diff
@@ -507,14 +511,6 @@ def __init__(self, secs=None, nrn_region=None, geometry=None, dimension=None, dx
             import neuron
             neuron.rxd.set_solve_type(secs, dimension=dimension)
         self._name = name
-        if dx is not None:
```

Comment:
this should not be deleted. This was a recent addition that gave more meaningful error messages